content/sensu-core/0.29/guides/pre-compile-plugins.md

```diff
@@ -44,27 +39,28 @@ gpgcheck=0
 enabled=1' | sudo tee /etc/yum.repos.d/sensu.repo && yum install -y sensu
```

Comment:
This needs a `sudo` on the `yum` command, as `&&` doesn't bring it with the previous `sudo tee` command.